src/views/studio/MyUploadsView/UploadStatusGroup/UploadStatusGroup.tsx

```diff
@@ -85,6 +85,9 @@ export const UploadStatusGroup: React.FC<UploadStatusGroupProps> = ({ uploads, s
   }, [errorsCount, hasUploadingAsset, isCompleted, isProcessing, missingAssetsCount])
 
   const renderAssetsGroupInfo = () => {
+    if (isWaiting) {
+      return 'Waiting for upload...'
```

Comment:
This copy sounds to me like I'm supposed to do something as a user. Maybe instead we could use "Starting upload..."? Is this used only when we're computing the hash of the asset or in some other case as well?